python/pyspark/sql/tests/test_udtf.py

```diff
@@ -3490,6 +3490,41 @@ def eval(self):
                     udtf(TestUDTF, returnType=ret_type)().collect()
 
 
+def test_udtf_with_collated_string_types(self):
```

Comment:
@ilicmarkodb the indent here is wrong, this test is actually skipped. It should be put into class `UDTFArrowTestsMixin`